malfeasance/handler.go

```diff
@@ -72,7 +75,7 @@ func (h *Handler) HandleMalfeasanceProof(ctx context.Context, peer p2p.Peer, dat
 		updateMetrics(p.Proof)
 		return nil
 	}
-	return ValidateAndSave(ctx, h.logger, h.cdb, h.cdb, h.edVerifier, h.cp, &p)
```

Comment:
if you do it report malfeasance to tortoise in `HandleSyncedMalfeasanceProof` and `HandleMalfeasanceProof` (before the self check), then malfeasance proofs reported by hare would also be captured.